spec/models/account_moderation_note_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe AccountModerationNote do
+  describe 'latest scope' do
+    it 'returns account moderation notes oldest to newest' do
+      account = Fabricate(:account)
+      note1 = Fabricate(:account_moderation_note, target_account: account)
+      note2 = Fabricate(:account_moderation_note, target_account: account)
+
+      expect(account.targeted_moderation_notes.latest).to eq [note1, note2]
```

Comment:
Both of these coverage additions here are good/useful ... odd that we were missing these. I bet there are more uncovered models which could use a once over to add simple model specs like this for validations/scopes.

For this one specifically - no real comments other than repeating from above re: `latest` scope name.

Possible future refactor (leaving as note to myself) - in the event something like https://github.com/mastodon/mastodon/pull/31013 merges, there are a ton of model specs whose validation assertions can be massively simplified.